css/src/style.scss

```diff
@@ -20,101 +20,75 @@ $white: #fff;
 	*/
```

Comment:
I think `$blue_background` should also be deprecated now - it was primarily used for *pickers, now it's only used to highlight input fields (in a way that doesn't seem needed or useful).

Reply:
Good idea, the border color should be handled by core by now.

Reply:
The box shadow was used on the input fields in the detail view. It made look a little bit more "fancy", but I am ok with removing them. --> Solved.